input/pagecontent/mcode-patient-bundle.md

```diff
@@ -0,0 +1,104 @@
+
+Note that these elements have a minimum [cardinality](https://www.hl7.org/fhir/conformance-rules.html#cardinality) of `0` **only** to allow them to be unreported if the Sender does not have the necessary data to generate a resource for a particular patient. If the Sender has the data, they SHALL be populated despite this.
+
+## Optional Elements
+
+mCODE does not exclude other elements in the mCODE Bundle. Senders MAY include the following if available:
+
+* Clinical notes, represented as `DiagnosticReport` or `DocumentReference` resources [as described by US Core][us-core-clinical-notes].
+* Diagnostic imaging related to a primary or secondary cancer condition, represented as [`Media`] or [`ImagingStudy`] resources.
+* `FamilyMemberHistory` resources that may be related to a primary or secondary cancer condition.
+* `Observation` resources that related to the patient or cancer diagnosis. For example, an `Observation` conforming to [`USCoreSmokingStatusProfile`] to indicate smoking status or an [an `Observation` indicating negative margins] on an excised tumor could be included.
+* `MedicationRequest` resources that do not conform to `CancerRelatedMedicationRequest`. Concomitant medications may be relevant in the cancer-treatment context. If any such resources are included, ALL MedicationRequest resources associated with the patient SHALL be included to ensure the absence of a given medication from the Bundle indicates the patient did not receive it.
+
+## Specifying a time range
+
+By default, this Bundle includes ALL resources described above irrespective of time. For some types of resources, such as vital signs, this may include a very large number of resources.
+
+To avoid this, callers may specify the `date` parameter when requesting a patient's Bundle via the Sender's FHIR API. This operates according to the [FHIR search specification for the `date` parameter](http://hl7.org/fhir/R4/search.html#date): for example, `date=ge2020-01-01&date=le2020-09-01` could be used to request resources within the provided date range.
+
+If the `date` parameter is provided, the following resources SHALL be filtered accordingly:
+
+- Laboratory results (based on the value of `effective`; if `effectivePeriod` is provided, use the end of the `Period` for date comparisons)
+- Medications (based on the value of `MedicationRequest.dosageInstruction.timing.event`)
+- Vital signs (based on the value of `effective`; if `effectivePeriod` is provided, use the end of the `Period` for date comparisons)
+
+Other resources SHALL NOT be filtered based on the `date` parameter. This is because the behavior must be specified to avoid ambiguous behavior, which would make implementation more difficult with little practical benefit.
```

Comment:
I don't understand the justification here "the behavior must be specified to avoid ambiguous behavior" sounds like double-talk. Explain.

Reply:
What I mean is that resources may have multiple date fields, so unless our spec explicitly indicates which one to use for filtering the behavior of date filtering will be ambiguous.

The main goal of the date filtering in my mind is performance optimization: it avoids sending large amounts of "out of range" data over the wire that would be immediately discarded by the client anyway. Beyond that, I don't think there's a strong argument for date filtering server-side, especially because the optimal behavior of date filtering (i.e., which date field(s) to consider) may be use case-specific.